index.html

```diff
@@ -1287,13 +1243,25 @@
         reject <var>promise</var> with
         <code>"<a>NotSupportedError</a>"</code>, and abort these steps.
       </li>
-          <li>Let <var>adapter</var> be <code>null</code>.</li>
+      <li>
+        Let <var>adapter</var> be <code>null</code>.
+        <p class="note">
+          There is a different <code>NFCAdapter</code> instance per
+          <code>Window</code> object, whose
```

Comment:
This "whose" refers to the `NFCAdapter`, but I think you mean that the Window's visibility and focus state determine things.